packages/xo-server/src/api/vif.js

```diff
@@ -82,8 +82,8 @@ export async function set({
   const newIpAddresses = []
   {
     const { push } = newIpAddresses
-    push.apply(newIpAddresses, allowedIpv4Addresses || vif.allowedIpv4Addresses)
-    push.apply(newIpAddresses, allowedIpv6Addresses || vif.allowedIpv6Addresses)
+    push.apply(newIpAddresses, allowedIpv4Addresses)
```

Comment:
If we change the VIF mac address, the IPs will keep existing in the network.

Reply:
Which IPs? Do you mean the allowed IPs?

Reply:
Yes allowed IPs

Reply:
Yes, we need to keep the IP addresses, except when the network is changing.